common-npm-packages/artifacts-common/module.json

```
@@ -16,6 +16,7 @@
         "ErrorInSettingUpSubscription": "Error in setting up subscription",
         "Error_FederatedTokenAquisitionFailed": "Failed to aquire federated token.",
         "Error_GetFeedTenantIdFailed": "Failed to find tenantId for feedUrl. %s",
+        "Error_InternalFeedsNotSupported": "Internal feeds are not supported for use with 'Azure Devops' Service Connections. To use an Azure Devops Service Connection, feedUrl must belong to an external organization.",
```

small suggestion to clarify the ado service connection CAN be used internally, just not with feed url

```suggestion
        "Error_InternalFeedsNotSupported": "FeedUrl must belong to an external organization. To authenticate to a feed in your organization with a 'Azure DevOps' service connection type, remove the `feedUrl` input",
```

---

slightly odd its in the common repo not the task but if its not possible/easy to move the task repo, please dont let this block

---

I agree its a bit odd, Ideally we'd be able to catch specific exceptions from the 
task itself but IIRC, that is more work than its worth. I'd say we can leave it here